packages/experimental/Button/src/CompoundButton/CompoundButtonFontTokens.ts

```diff
@@ -0,0 +1,24 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { CompoundButtonTokens } from './CompoundButton.types';
+
+export const defaultCompoundButtonFontTokens: TokenSettings<CompoundButtonTokens, Theme> = (_t: Theme): CompoundButtonTokens => ({
+  medium: {
+    hasContent: {
+      variant: 'bodySemibold',
```

Comment:
Nit: Can these reference global / alias tokens? Or are we keeping that in just the win32 and maybe macos files for now?

Reply:
I was going to fix up the text tokens later, since I haven't fully worked them out yet.

Reply:
Could be considered part of #1323

Reply:
Sounds fair!